worker/study_tts_worker/worker.py

```
@@ -230,65 +283,562 @@ def _apply_offline_environment(launcher: dict[str, Any]) -> None:
+        )
+
+    # Loaded per profile so the conditioning reported above is conditioning this
+    # process actually holds, rather than a record it read and could not use.
+    conditionals: dict[str, Any] = {}
+    try:
+        for identity in conditioning:
+            conditionals[identity] = Conditionals.load(
+                str(voice_root / identity / "conditionals.pt"), map_location="cpu"
+            ).to("cpu")
+    except Exception as error:  # noqa: BLE001 - any backend fault is one refusal
+        raise BackendUnavailable(f"a voice conditioning artifact could not be loaded: {_redacted_detail(error)}")
```

Ruff (0.16.2)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)